db/schema.rb

```diff
@@ -0,0 +1,58 @@
+# encoding: UTF-8
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20160427220429) do
+
+  create_table "markets", force: :cascade do |t|
+    t.integer  "market_id"
+    t.text     "name",       null: false
+    t.text     "address"
+    t.text     "city"
```

Comment:
It's interesting that you chose to use the text instead of string data type for these. Generally text is used for longer textual data like paragraphs while string is used for shorter things like "title" etc. and has a character limit of 255. Doing a quick google on which is recommended to use some people say the smaller the field size the more efficient the database, while other's say there is no significant performance difference soo ¯\_(ツ)_/¯ I didn't go too in depth searching that though, so it's something you could look further into if you felt like.